### PR TITLE
EDM-366: Improve token request URL for OCP auth

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: internal/cli/login.go [hunk 1]
```diff
@@ -276,17 +298,23 @@ func (o *LoginOptions) Run(ctx context.Context, args []string) error {
...
 				clientId = o.ClientId
 			}
-			token, err = o.getOauth2Token(fmt.Sprintf("%s/.well-known/oauth-authorization-server", resp.JSON200.AuthURL), clientId, "")
+			token, err = o.getOauth2Token(oauthConfigUrl, clientId, "")
 			if err != nil {
 				return err
 			}
 		} else if o.Token == "" {
-			fmt.Printf("You must obtain an API token by visiting %s\n", resp.JSON200.AuthURL)
+			oathConfig, err := o.getOauthConfig(oauthConfigUrl)
```
Comment (mangelajo) [inline internal/cli/login.go:312]:
```suggestion
			oauthConfig, err := o.getOauthConfig(oauthConfigUrl)
```

File: internal/cli/login.go [hunk 2]
```diff
@@ -276,17 +298,23 @@ func (o *LoginOptions) Run(ctx context.Context, args []string) error {
...
+			if err != nil {
+				return fmt.Errorf("could not get oauth config: %w", err)
+			}
+
+			fmt.Printf("You must obtain an API token by visiting %s/request\n", oathConfig.TokenEndpoint)
```
Comment (mangelajo) [inline internal/cli/login.go:317]:
```suggestion
			fmt.Printf("You must obtain an API token by visiting %s/request\n", oauthConfig.TokenEndpoint)
```

**Review by mangelajo — COMMENTED** [review]
A nit, but it looks good, nice improvement.

**Comment by rawagner** [comment]
Thank you @mangelajo . I fixed the typo :)

**Review by mangelajo — APPROVED** [review]

**Pull request merged**